cni/log/logger.go

```diff
@@ -17,9 +19,10 @@ var (
 const (
 	maxLogFileSizeInMb = 5
 	maxLogFileCount    = 8
+	etwCNIEventName    = "Azure-CNI"
 )
 
-func initZapLog(logFile string) *zap.Logger {
+func initZapLog(logFile string, isEtwLoggingEnabled bool) *zap.Logger {
```

Comment:
I'm not crazy about bool options because they defy documentation. I think I'd rather this be a separate function with the common initialization factored out to a common func.